functional-annotation.wdl

```diff
@@ -1,50 +1,51 @@
+version 1.0
 workflow f_annotate {
-  String  imgap_project_id
-  String  imgap_project_type
-  Int     additional_threads
-  File?    input_contigs_fasta
-  File    input_fasta
-  String  database_location
-  Boolean ko_ec_execute=true
-  String  ko_ec_img_nr_db="${database_location}"+"/IMG-NR/20230629/img_nr"
-  String  ko_ec_md5_mapping="${database_location}"+"/IMG-NR/20230629/md5Hash2Data.tsv"
-  String  ko_ec_taxon_to_phylo_mapping="${database_location}"+"/IMG-NR/20230629/taxonOId2Taxonomy.tsv"
-  String  lastal_bin="/opt/omics/bin/lastal"
-  String  selector_bin="/opt/omics/bin/functional_annotation/lastal_img_nr_ko_ec_gene_phylo_hit_selector.py"
-  Boolean smart_execute=true
-  Int?    par_hmm_inst
-  Int?    approx_num_proteins
-  String  smart_db="${database_location}"+"/SMART/01_06_2016/SMART.hmm"
-  String  hmmsearch_bin="/opt/omics/bin/hmmsearch"
-  String  frag_hits_filter_bin="/opt/omics/bin/functional_annotation/hmmsearch_fragmented_hits_filter.py"
-  Boolean cog_execute=true
-  String  cog_db="${database_location}"+"/COG/HMMs/2003/COG.hmm"
-  Boolean tigrfam_execute=true
-  String  tigrfam_db="${database_location}"+"/TIGRFAM/v15.0/TIGRFAM.hmm"
-  String  hit_selector_bin="/opt/omics/bin/functional_annotation/hmmsearch_hit_selector.py"
-  Boolean superfam_execute=true
-  String  superfam_db="${database_location}"+"/SuperFamily/v1.75/supfam.hmm"
-  Boolean pfam_execute=true
-  String  pfam_db="${database_location}"+"/Pfam/Pfam-A/v34.0/Pfam-A.v34.0.hmm"
-  String  pfam_claninfo_tsv="${database_location}"+"/Pfam/Pfam-A/v34.0/Pfam-A.clans.tsv"
-  String  pfam_clan_filter="/opt/omics/bin/functional_annotation/pfam_clan_filter.py"
-  Boolean cath_funfam_execute=true
-  String  cath_funfam_db="${database_location}"+"/Cath-FunFam/v4.2.0/funfam.hmm"
-#  Boolean signalp_execute=true
-#  String  signalp_gram_stain="GRAM_STAIN"
-#  String  signalp_bin="/opt/omics/bin/signalp"
-#  Boolean tmhmm_execute=true
-#  String  tmhmm_model="/opt/omics/programs/tmhmm-2.0c/lib/TMHMM2.0.model"
-#  String  tmhmm_decode="/opt/omics/bin/decodeanhmm"
-#  String  tmhmm_decode_parser="/opt/omics/bin/functional_annotation/decodeanhmm_parser.py"
-  File    sa_gff
-  String  product_assign_bin="/opt/omics/bin/functional_annotation/assign_product_names_and_create_fa_gff.py"
-  String  product_names_mapping_dir="${database_location}"+"/Product_Name_Mappings/20230814"
-  String  container
-  String  hmm_container="microbiomedata/img-omics@sha256:d5f4306bf36a97d55a3710280b940b89d7d4aca76a343e75b0e250734bc82b71"
-  String  last_container="microbiomedata/img-omics@sha256:d5f4306bf36a97d55a3710280b940b89d7d4aca76a343e75b0e250734bc82b71"
-
-  if(ko_ec_execute) {
+    input {
+        String  imgap_project_id
+        String  imgap_project_type
+        Int     additional_threads
+       # File    input_contigs_fasta
+        File    input_fasta
+        String  database_location
```

Comment:
default database location here should be /refdata to be consistent with other WDLs